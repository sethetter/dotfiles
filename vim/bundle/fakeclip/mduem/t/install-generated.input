#!/bin/bash

dir='t/,install-generated'
makefile='t/,install-generated.Makefile'
file='t/,install-generated.foo'

cat >"$makefile" <<END
TARGETS_GENERATED := $file

$file:
	echo $$ >\$@

include Makefile
END

make --always-make -f "$makefile" >/dev/null \
  "INSTALLATION_DIR=$dir" \
  'TARGETS_STATIC=Makefile README' \
  install

find "$dir"

rm -rf "$dir" "$makefile" "$file"

# __END__
