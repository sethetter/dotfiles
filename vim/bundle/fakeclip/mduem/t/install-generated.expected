t/,install-generated
t/,install-generated/Makefile
t/,install-generated/README
t/,install-generated/t
t/,install-generated/t/,install-generated.foo
